src/flap.sol

```diff
@@ -143,9 +154,11 @@ contract Flapper is LibNote {
     function deal(uint id) external note {
         require(live == 1, "Flapper/not-live");
         require(bids[id].tic != 0 && (bids[id].tic < now || bids[id].end < now), "Flapper/not-finished");
-        vat.move(address(this), bids[id].guy, bids[id].lot);
+        uint256 lot = bids[id].lot;
+        vat.move(address(this), bids[id].guy, lot);
         gem.burn(address(this), bids[id].bid);
         delete bids[id];
+        usage = sub(usage, lot);
```

Comment:
Something for the team to think about is if we can ever get into a situation where `usage` is lower than `lot`. I don't see anything immediately but we will freeze up auctions if this gets mismatched.

Reply:
Yeah trying to think if it's worthwhile adding a `file` for usage but I don't think it is because people will be tempted to use it and that will really mess things up. In the unexpected case this does happen we can just eat the loss on the particular action and deploy a new flap.

Reply:
I added the conditional.

Reply:
Actually, this conditional feels weird and a waste of gas. I think I'll leave it out.